flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/CatalogManager.java

```diff
@@ -656,8 +663,16 @@ public ObjectIdentifier qualifyIdentifier(UnresolvedIdentifier identifier) {
     public void createTable(
             CatalogBaseTable table, ObjectIdentifier objectIdentifier, boolean ignoreIfExists) {
         execute(
-                (catalog, path) ->
-                        catalog.createTable(path, resolveCatalogBaseTable(table), ignoreIfExists),
+                (catalog, path) -> {
+                    ResolvedCatalogBaseTable<?> resolvedTable =
+                            managedTableListener.notifyTableCreation(
+                                    catalog,
+                                    objectIdentifier,
+                                    resolveCatalogBaseTable(table),
```

Comment:
nit: introduce a local variable here and below, it avoids too much nesting and makes it easier to set breakpoints during debugging